app/controllers/gift_requests_controller.rb

```diff
@@ -18,6 +18,8 @@ def edit
   end
 
   def create
+    @back_route = wishlist_path(params[:id])
```

Comment:
create is a POST request and shouldn't have a view, so this should be unnecessary